package.json

```diff
@@ -127,7 +127,7 @@
     "postcss-loader": "3.0.0",
     "promise": "8.0.3",
     "raf": "3.4.1",
-    "react-dev-utils": "^10.1.0",
+    "react-dev-utils": "^11.0.4",
```

Comment:
For some reason this update causes the icons library to throw an error and not work as expected. This will require a deeper investigation to update this without issues